.github/workflows/build-master.yml

```diff
@@ -0,0 +1,57 @@
+name: Build
+
+on:
+  repository_dispatch:
+    types: [stdlib-publish-snapshot]
+  workflow_dispatch:
+  push:
+    branches:
+      - master
```

Comment:
this should be `main`, check all places

Reply:
refer: refer : https://github.com/ballerina-platform/module-ballerina-random/tree/main/.github/workflows

Reply:
Fixed with [`b692bc0`](https://github.com/ballerina-platform/module-ballerinax-mssql/pull/2/commits/b692bc0d7b0816ee9fdf8712a18ce93c31ce5819)